config/web/settings.ini

```diff
@@ -39,7 +39,7 @@ web.https.port=8443
 
 # scheduler user that is used as cache
 scheduler.cache.login=watcher
-scheduler.cache.password=w_pwd
+scheduler.cache.password=watcher_pwd
```

Comment:
it would be best to create as well a credential file for the watcher account, to remove the scheduler.cache.password, and to use the scheduler.cache.credential property instead. I just hope that this works with relative paths from the scheduler home.